packages/lisk-transactions/src/4_register_multisignature_account.js

```diff
@@ -12,16 +12,52 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { MULTISIGNATURE_FEE } from './constants';
+import {
+	MULTISIGNATURE_FEE,
+	MULTISIGNATURE_MAX_LIFETIME,
+	MULTISIGNATURE_MIN_LIFETIME,
+	MULTISIGNATURE_MAX_SIGNATURES,
+	MULTISIGNATURE_MIN_SIGNATURES,
+} from './constants';
 import {
 	prependPlusToPublicKeys,
 	validateKeysgroup,
 	wrapTransactionCreator,
+	isValidInteger,
 } from './utils';
 
-const registerMultisignatureAccount = ({ keysgroup, lifetime, minimum }) => {
+const validateInputs = ({ keysgroup, lifetime, minimum }) => {
+	if (
+		!isValidInteger(lifetime) ||
+		lifetime < MULTISIGNATURE_MIN_LIFETIME ||
+		lifetime > MULTISIGNATURE_MAX_LIFETIME
+	) {
+		throw new Error(
+			`Please provide a valid lifetime value. Expected integer between ${MULTISIGNATURE_MIN_LIFETIME} and ${MULTISIGNATURE_MAX_LIFETIME}.`,
+		);
+	}
+
+	if (
+		!isValidInteger(minimum) ||
+		minimum < MULTISIGNATURE_MIN_SIGNATURES ||
+		minimum > MULTISIGNATURE_MAX_SIGNATURES
+	) {
+		throw new Error(
+			`Please provide a valid minimum value. Expected integer between ${MULTISIGNATURE_MIN_SIGNATURES} and ${MULTISIGNATURE_MAX_SIGNATURES}.`,
+		);
+	}
+
+	if (keysgroup.length < minimum) {
```

Comment:
@shuse2 This should be `keysgroup.length < minimum - 1`, right?

Reply:
Yes!
On the other PR, i fixed it on schema check in opposite way (but same thing)
https://github.com/LiskHQ/lisk-elements/pull/770/files#diff-bc8696a3bc628a23ff3642d6f438a7e5R37

Reply:
https://github.com/LiskHQ/lisk/blob/development/logic/multisignature.js#L135-L136
This is correct according to the `core`